docs/fault-proof-alpha/immunefi.md

```diff
@@ -0,0 +1,90 @@
+# Fault Proof Alpha Bounty
+
+The fault proof alpha system will be deployed to the Goerli testnet as a sidecar to the current system. During the alpha, the outcome of fault dispute games will have no influence on the official bridge contracts nor the official `L2OutputOracle`.
+During this early phase of ongoing development, we invite security researchers and developers to engage with the system and attempt to break its current components.
+
+The current system is not production ready, however the core infrastructure for creating an instruction trace ([Cannon][cannon] + the [`op-program`][op-program]), the off-chain challenge agent ([`op-challenger`][op-challenger]),
+and the on-chain infrastructure for the [Dispute Game][dispute-game] are all in place.
+
+### Resources
+
+> **Note** 
+> Prior to moving forward, we recommended reading into the technical documentation for the components of Fault Proof Alpha.
+
+* [Cannon][cannon] & [Cannon Contracts][cannon-contracts]
+    * [Cannon VM Specs][cannon-vm-specs]
+* [`op-program`][op-program]
+    * [Fault Proof Specs][fault-proof-specs]
+* [Dispute Game][dispute-game]
+    * [Fault Dispute Game Specs][fault-dispute-specs]
+* [`op-challenger`][op-challenger]
+
+## Known Issues
+The alpha system is not prepared for mainnet, and as such, there are a number of known issues that we are working on fixing and components of the system that must be improved prior to it being sustainable.
+
+1. DoS attacks are currently likely to occur due to the lack of bonds in the alpha system as well as the lack of an extra layer of bisection in the dispute game to reduce the running time of [Cannon][cannon]. It is possible to
+    DoS the network of honest challengers by creating a large number of invalid challenges.
+
+### Reviewer Notes
+1. **Any bug report without a PoC in the form of a test in `op-e2e` will not be considered a valid bug report.**
```

Comment:
Requiring a test to validate the bug report is good, but the chain-parameters need to be better defined. Op-e2e does not quite run the mainnet configuration already (shorter output root interval, different block time, etc.). Changing the parameters wildly may create undefined behavior. I listed it in the known issues, but we still have to better define the bounds on what counts as a valid chain to break a fault proof in.

Reply:
This is a great point. Will cover this in the meeting tonight!